packages/lambda-send-payment/send-email.js

```diff
@@ -127,15 +126,9 @@ ${displayItems(lineItems, '£', '\r\n')}
 
 Total value: £${amount}
 
-The address you submitted was:
-
-${
-  [(`${firstName} ${lastName}`).trim(), address1, address2, address3, postCode].filter(Boolean).join('\r\n')
-}
-
 Your order will be submitted to the factory at the end of the month.
 
-Once we know the delivery date for your items we'll let you know.
+Typically it takes about six weeks to print, after which your items will be held at Rat Race Cycles for you to pick up.
```

Comment:
Not sure how I feel about putting the six weeks bit in, but that does seem like a useful piece of info to include.

Reply:
It's good to address people's expectations.